etc/adminhtml/system.xml

```diff
@@ -4,5 +4,21 @@
         <tab id="extensions_list" translate="label" sortOrder="999999">
             <label>Extensions</label>
         </tab>
+        <section id="extensions_list_settings" translate="label" showInDefault="1" showInWebsite="0" showInStore="0" sortOrder="1">
+            <tab>extensions_list</tab>
+            <label>Settings</label>
+            <resource>RedChamps_CleanMenu::extensions_settings</resource>
+            <group id="general" translate="label" showInDefault="1" showInWebsite="0" showInStore="0" sortOrder="10">
+                <label>General</label>
+                <field id="allowed_modules" type="multiselect" translate="label comment" showInDefault="1" showInWebsite="0" showInStore="0" sortOrder="10" canRestore="1">
+                    <label>Allowed Modules</label>
+                    <source_model>RedChamps\CleanMenu\Model\Config\Source\Modules</source_model>
+                    <backend_model>RedChamps\CleanMenu\Model\Config\Backend\Modules</backend_model>
+                    <can_be_empty>1</can_be_empty>
+                    <config_path>extensions_list_settings/general/allowed_modules</config_path>
+                    <comment>We will allow the selected modules to have their own menus and tabs in the config section.</comment>
```

Comment:
I think this comment needs to be edited

Reply:
Feel free to suggest comment

Reply:
I think this should be enough "We will allow the selected modules to have their own menus."

Reply:
I've added the possibility to allow specific custom tabs

Reply:
Cool! Not sure, but will it be better if we show tab labels instead of code for the values of this setting?

![allowed_tabs](https://user-images.githubusercontent.com/5336201/72330109-f38c0900-36db-11ea-9669-4437549218b5.png)

Reply:
Good catch, it has been fixed

Reply:
Awesome! All looks good now.